<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>674</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>870</width>
    <height>674</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>870</width>
    <height>674</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Aplicatie de tip Zero-config bazata pe mDNS si DNS-SD</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>870</width>
     <height>630</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>871</width>
     <height>631</height>
    </size>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-61</x>
      <y>-1</y>
      <width>941</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(52, 101, 164)</string>
    </property>
    <widget class="QListView" name="serviceTypesBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTipDuration">
      <number>-6</number>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(238, 238, 236);</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="outputDisplay">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>481</width>
       <height>531</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <stylestrategy>NoAntialias</stylestrategy>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 0, 60);
color: rgb(252, 233, 79);
</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>**************************RESULTS*************************</string>
     </property>
     <property name="overwriteMode">
      <bool>true</bool>
     </property>
     <property name="centerOnScroll">
      <bool>false</bool>
     </property>
     <property name="textColor" stdset="0">
      <color>
       <red>252</red>
       <green>233</green>
       <blue>79</blue>
      </color>
     </property>
    </widget>
    <widget class="QLabel" name="labelServiceTypes">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>ServiceTypes</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>0</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>338</x>
       <y>50</y>
       <width>91</width>
       <height>161</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>SEARCH</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>0</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelOutput">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>0</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Output</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>0</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="ipAddress">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>590</y>
       <width>311</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(211, 215, 207);</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>560</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelIpAddress">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>560</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>  IP ADDRESS</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>560</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelHostName">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>  HostName</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="hostName">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(211, 215, 207);</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_13">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_14">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>199</y>
       <width>20</width>
       <height>61</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelOldHost">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>440</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>OLD HostName</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="oldHostName">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>470</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(211, 215, 207);</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_15">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>420</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_16">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>420</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelNewHost">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>500</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>  NEW HostName</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="newHostName">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>530</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(211, 215, 207);</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_17">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>490</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_18">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>490</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelService">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>330</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>        Service</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="serviceName">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>360</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(211, 215, 207);</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_19">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>330</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_20">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>330</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="getIPButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>290</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Get IP Address</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addServiceButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>390</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeServiceButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>390</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>REMOVE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editHostNameButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>570</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>EDIT</string>
     </property>
    </widget>
    <zorder>line_17</zorder>
    <zorder>line_18</zorder>
    <zorder>labelNewHost</zorder>
    <zorder>line_15</zorder>
    <zorder>line_16</zorder>
    <zorder>line_19</zorder>
    <zorder>line_20</zorder>
    <zorder>line_13</zorder>
    <zorder>line_14</zorder>
    <zorder>line_5</zorder>
    <zorder>line_6</zorder>
    <zorder>line_3</zorder>
    <zorder>line_4</zorder>
    <zorder>line_2</zorder>
    <zorder>line</zorder>
    <zorder>serviceTypesBox</zorder>
    <zorder>outputDisplay</zorder>
    <zorder>labelServiceTypes</zorder>
    <zorder>searchButton</zorder>
    <zorder>labelOutput</zorder>
    <zorder>ipAddress</zorder>
    <zorder>labelIpAddress</zorder>
    <zorder>labelHostName</zorder>
    <zorder>hostName</zorder>
    <zorder>labelOldHost</zorder>
    <zorder>oldHostName</zorder>
    <zorder>newHostName</zorder>
    <zorder>labelService</zorder>
    <zorder>serviceName</zorder>
    <zorder>getIPButton</zorder>
    <zorder>addServiceButton</zorder>
    <zorder>removeServiceButton</zorder>
    <zorder>editHostNameButton</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEXIT">
    <property name="title">
     <string>EXIT</string>
    </property>
   </widget>
   <addaction name="menuAbout"/>
   <addaction name="menuEXIT"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
